docs/basic-docs/atomicdex/atomicdex-tutorials/how-to-become-a-liquidity-provider.md

```diff
@@ -20,240 +20,29 @@ If the user prefers to use computer hardware at home, instead of a VPS, we recom
 - RAM - _at least_ 4 GB
 - Processor - _at least_ i5 or equivalent
 
-## Installing Dependencies
+## Get the MarketMaker and  Coins Configuration File
 
-#### Step 1: OS Packages
+#### Step 1: Get the MarketMaker Maker
 
-##### Command
-
-```bash
-sudo apt update
-sudo apt-get install build-essential git jq llvm-3.9-dev libclang-3.9-dev clang-3.9 cmake libssl-dev pkg-config
-```
+You can get the MarketMaker by using builds in the repository or by building from source.
```

Comment:
we call this software "AtomicDEX-API" if we'll call it differently in different places (e.g. MarketMaker in this case) it'll create a confusion

Reply:
I checked the previous version and found that it didn't use "AtomicDEX-API" except for directory name. It is just "MarketMaker". Are you refereeing to other tutorial or documents? I suggest fixing that issue in a later PR as I see there are other tutorials using "MarketMaker"

Reply:
I will correct this in all guides